src/components/MainPage/MainPage.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { Story } from '@storybook/react'
+import MainPage from './MainPage'
+
+export default {
+  title: 'MainPage',
+  component: MainPage,
+}
```

Comment:
```suggestion
} as Meta;
```


With: `import { Story, Meta } from '@storybook/react'`